### PR TITLE
Add all mentioned ignored files across libraries

### PR DESCRIPTION
**Pull request by tekktrik**
Behold, the monster `.gitgnore` list!  I went through and cleaned up duplicates (like `build` vs `build/` (kept former)) but this is everything across the Official Adafruit Bundle in all the `.gitignore` files!

If there's any duplicates I forgot, or anything that should be added, please suggest it!

### CODE REVIEW

**Comment by kattni** [comment]
I requested this be put together based on the content in any of the libraries, so we can create a canonical `.gitignore` file to then apply across all the libs. This is much bigger than I expected, so it's definitely worth doing some cleanup. I plan to go through it thoroughly tomorrow.

Regardless, the `.gitignore` file should be identical in every library. I want this to be what we apply to all of the libs when it's merged.

File: {{ cookiecutter and 'tmp_repo' }}/.gitignore [hunk 1]
```diff
@@ -2,17 +2,152 @@
...
+*.pot
+*.py,cover
+*.py[cod]
 *.pyc
+*.sage.py
+*.so
+*.spec
+*.stackdump
+*.swp
+*.ttf
```
Comment (dhalbert) [inline {{ cookiecutter and 'tmp_repo' }}/.gitignore:33]: `.ttf's` are font files and could conceivably be in a library.

File: {{ cookiecutter and 'tmp_repo' }}/.gitignore [hunk 2]
```diff
@@ -2,17 +2,152 @@
...
-_build
+*.msi
+*.msix
+*.msm
+*.msp
+*.orig
+*.pot
+*.py,cover
+*.py[cod]
 *.pyc
```
Comment (jepler) [inline {{ cookiecutter and 'tmp_repo' }}/.gitignore:27]: .pyc files are covered twice with this addition

File: {{ cookiecutter and 'tmp_repo' }}/.gitignore [hunk 3]
```diff
@@ -2,17 +2,152 @@
...
+*.swp
+*.ttf
+*.vsix
+*notes*
+*~
+.AppleDB
+.AppleDesktop
+.AppleDouble
+.DS_STORE
+.DS_Store
```
Comment (jepler) [inline {{ cookiecutter and 'tmp_repo' }}/.gitignore:41]: Also added at line 8

**Review by jepler — COMMENTED** [review]
Consider sorting this file so that any duplicates are easier to spot.

**Comment by kattni** [comment]
I'm creating a new `.gitignore` to begin from. This version is not viable to work with to generate what will be a minimal CircuitPython-library-specific list. I will submit a PR here when I have something, and we can begin working from there.

**Pull request closed without merging**